docs/CHANGELOG.rst

```diff
@@ -12,8 +12,11 @@ Unreleased
 
 Changed
 -------
-- Bump MultiQC to 1.7.0 and add QoRTs (1.3.0) package to
-  ``resolwebio/common:1.1.0`` Docker image
+- Update ``resolwebio/common:1.1.0`` Docker image:
```

Comment:
@JureZmrzlikar `resolwebio/common:1.1.0` is currently unused, so I decided to re-use the tag.